server/server.js

```diff
@@ -224,8 +224,13 @@ app.post('/getHotAsks', function (req, res) {
   if (batch.hotAsks !== undefined && batch.hotAsks.Items.length > 0) {
     console.log("batch.hotAsks data exist, use this");
     batch.hotAsks.Items[0]['rank'] = 1;
-    batch.hotAsks.Items[1]['rank'] = 2;
-    batch.hotAsks.Items[2]['rank'] = 3;
+
+    if(batch.hotAsks.Items.length >= 2)
+      batch.hotAsks.Items[1]['rank'] = 2;
+
+    if(batch.hotAsks.Items.length >= 3)
+      batch.hotAsks.Items[2]['rank'] = 3;
+
```

Comment:
How to use 'for' loop for future expansion rank

(ex)
for (var i = 0; i < batch.hotAsks.Items.length; i ++) {
  batch.hotAsks.Items[i][‘rank’] = i + 1;

  if (i === 2) {
    break;
  }
}